### ISSUE TITLE
Overgrown coop disassemble machines arms aren't using rusty textures.

### ISSUE THREAD

**Issue by ZombiePwner1** [BEEmod/BEE2-items#1824]
![20170408123113_1](https://cloud.githubusercontent.com/assets/18079316/24825727/60e2be88-1c57-11e7-9b36-f2aa08231ccb.jpg)

![20170408123118_1](https://cloud.githubusercontent.com/assets/18079316/24825728/62cf06a2-1c57-11e7-8ed7-eebd65f6b7a7.jpg)

**Comment by Luke18033** [comment]
I personally don't like the rusty disassemblers or gel droppers, I think they would look better if they were just a little dirty.

**Comment by ZombiePwner1** [comment]
What about the catwalks?

**Comment by Luke18033** [comment]
Those and the wallpipes look great.

**Comment by Luke18033** [comment]
Yeah I don't think the rusted look really works for anything with color. There are a few like "dirt" detail textures, maybe use one of those instead. Then the disassemble arms wouldn't look as out of place.
